com.unity.netcode.gameobjects/Components/NetworkAnimator.cs

```diff
@@ -133,7 +133,7 @@ internal void ResetParameterOptions()
             m_ParameterSendBits = 0;
         }
 
-        private bool sendMessagesAllowed
+        private bool SendMessagesAllowed
```

Comment:
This could be expensive in the long run since it is accessed in the FixedUpdate.

Reply:
Good idea, change made